README.md

```diff
@@ -31,7 +31,7 @@ it's own sandboxed workspace.
 * [Using a http proxy](docs/USING-A-HTTP-PROXY.md)
 * [Prelaunch JUT](docs/PRELAUNCH.md)
 * [Obtaining a report of plugins that were exercised](docs/EXERCISEDPLUGINSREPORTER.md)
-* [Managing the versions of Jenkins and plugins](docs/SUT-VERSIONS.md)
+* [Managing the versions of Jenkins and plugins](docs/SUT-VERSIONS.md). This includes things like running the ATH with a local build of a plugin.
```

Comment:
If you are trying to debug you will want to know how to... I spent ages before I found that page